### ISSUE TITLE
Unable to access the web app 

### ISSUE THREAD

**Issue by Sdelausnay** [microsoft/chat-copilot#931]
**Describe the bug**
I have follow the ReadMe to deploy the solution to Azure. Service are all created and can deploy the webapp. When I navigate to the Chat Copilot, I get the request to login and everything seem ok for the login process. But then it got stuck on the Connecting page. Doesn't get any further, looks like he is waiting on the backend to respond. 

Only thing I changed to the deployment, is deploy everything to SwedenCentral. 

Only thing I cloud find in the logs is a File not found but no details why it is not there. 

`Could not load file or assembly 'StartupBootstrapper, Culture=neutral, PublicKeyToken=null'. The system cannot find the file specified.`

**To Reproduce**
1. Followed the Read Me in scripts/deploy
2. Changed the location in deploy-azrue.ps1 to Sweden Central
3. Run the deploy-azure.ps1 script
4. Run package-webapi.ps1
5. Run deploy-webapi.ps1
6. Access the webapp and login

**Expected behavior**
The application moves to the chat windows after login. 

**Screenshots**
![image](https://github.com/microsoft/chat-copilot/assets/62655804/0d201852-9662-41d4-bea5-ba95319f9763)

**Platform**
 - OS: Azure
 - IDE:  Deployed through VS Code
 - Language: C#

**Additional context**

**Comment by Sdelausnay** [comment]
Found some additional logs

```logs
2024-04-15T14:24:18  PID[5332] Information 04-15 14:24:18 Error: Unexpected exception in agent main process. Details: Microsoft.ServiceProfiler.Utilities.AppIdNotFoundException: Unable to find AppId for iKey:
2024-04-15T14:24:18  PID[5332] Information    at Microsoft.ServiceProfiler.Utilities.AppInsightsProfileFetcher.<FetchProfileAsync>d__3.MoveNext()
2024-04-15T14:24:18  PID[5332] Information --- End of stack trace from previous location where exception was thrown ---
2024-04-15T14:24:18  PID[5332] Information    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
2024-04-15T14:24:18  PID[5332] Information    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2024-04-15T14:24:18  PID[5332] Information    at Microsoft.ServiceProfiler.DiagnosticsAgent.Program.GetApplicationInsightsAppId(String instrumentationKey, String appInsightsEndpoint)
2024-04-15T14:24:18  PID[5332] Information    at Microsoft.ServiceProfiler.DiagnosticsAgent.Program.Main(String[] cmdArgs)
```

**Comment by Sdelausnay** [comment]
Seems to be a Front-end issue. In mean time recreated the App registration and also tried to run it locally with authentication and I'm getting the same error.  Adding some logs to the Frontend revelated following issue:

`Received error while retrieving access token: BrowserAuthError: no_account_error: No account object provided to acquireTokenSilent and no active account has been set. Please call setActiveAccount or provide an account on the request.`

**Comment by hsirtl** [comment]
Same issue here. Getting the "Connecting..." message both when running the app via docker compose and when running it on Azure (having installed it following the deploy-instructions).

**Comment by Sdelausnay** [comment]
I found the issue, seems the @azure/msal-browser version currently in the master branch is having the issue. 

Still further investigating, but when I update the package, seems to be working.

**Comment by glahaye** [comment]
I believe this is fixed now.  Can you give it another to confirm the fix works for you?

**Comment by glahaye** [comment]
Fixed by #969
